examples/gno.land/r/demo/gnofundme/gnofundme.gno

```diff
@@ -0,0 +1,319 @@
+package gnofundme
+
+import (
+	"bytes"
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+	GFMpkg "gno.land/p/demo/gnofundme" // GnoFundMe package
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	campaigns      *avl.Tree
+	users          *avl.Tree
+	banker         std.Banker
+	balance        int64
+	campaignPrefix = "/r/demo/gnofundme:"
+	campaignsTotal = make(map[string]int64)
+)
+
+func init() {
+	campaigns = avl.NewTree()
+	users = avl.NewTree()
+	banker = std.GetBanker(std.BankerTypeRealmIssue)
+	// banker.IssueCoin(std.CurrentRealm().Addr(), "ugnot", 350000000)
+}
+
+func NewUser(name string, email string) string {
```

Comment:
Wouldn't this then be `Register` or similar?

Not sure you need to have this whole user registration flow; you could just use the addresses as the primary point, where the campaign could have the owner address, name, email, etc, as optional.

Would be more simple that way.